test/create-websocket-stream.test.js

```diff
@@ -295,11 +295,14 @@ describe('createWebSocketStream', () => {
         ws._socket.write(Buffer.from([0x85, 0x00]));
       });
 
-      assert.strictEqual(process.listenerCount('uncaughtException'), 1);
+      assert.strictEqual(
+        process.listenerCount('uncaughtException'),
+        EventEmitter.usingDomains ? 2 : 1
+      );
 
-      const [listener] = process.listeners('uncaughtException');
+      const listener = process.listeners('uncaughtException').pop();
```

Comment:
The domain added `'uncaughtException'` listener is always prepended and kept, right?

Reply:
It is prepended when you add the first listener for `'uncaughtException'` and removed when you remove the last listener for it (excluding the one from `domain`)

```
> let foo = () => {}
> let bar = () => {}
> process.listenerCount('uncaughtException')
0
> process.on('uncaughtException', foo); process.listenerCount('uncaughtException')
2
> process.on('uncaughtException', bar); process.listenerCount('uncaughtException')
3
> process.off('uncaughtException', foo); process.listenerCount('uncaughtException')
2
> process.off('uncaughtException', bar); process.listenerCount('uncaughtException')
0
```